sonic-xcvrd/scripts/xcvrd

```diff
@@ -107,6 +107,27 @@ def beautify_dom_info_dict(dom_info_dict):
     dom_info_dict['tx3power'] = strip_unit_and_beautify(dom_info_dict['tx3power'], POWER_UNIT)
     dom_info_dict['tx4power'] = strip_unit_and_beautify(dom_info_dict['tx4power'], POWER_UNIT)
 
+def beautify_dom_threshold_info_dict(dom_info_dict):
```

Comment:
I would like to suggest adding these to "beautify_dom_threshold_info_dict", as I think they should be part of the DOM.